src/elements/element.rectangle.js

```diff
@@ -69,7 +71,7 @@ module.exports = Element.extend({
 			bottom = vm.base;
 			signX = 1;
 			signY = bottom > top ? 1 : -1;
-			borderSkipped = vm.borderSkipped || 'bottom';
+			borderSkipped = valueOrDefault(vm.borderSkipped, {bottom: true}) || {};
```

Comment:
When do we hit the `|| {}` case?

Reply:
`borderSkipped = false` or `borderSkipped = null`